data/json/recipes/armor/storage.json

```diff
@@ -305,6 +305,20 @@
     "qualities": [ { "id": "SEW", "level": 1 } ],
     "components": [ [ [ "leather", 6 ], [ "tanned_hide", 1 ] ], [ [ "filament", 300, "LIST" ], [ "cordage", 1, "LIST" ] ] ]
   },
+  {
+    "result": "stone_pouch",
+    "type": "recipe",
+    "category": "CC_ARMOR",
+    "subcategory": "CSC_ARMOR_STORAGE",
+    "skill_used": "tailor",
+    "difficulty": 3,
+    "skills_required": [ [ "throw", 1 ] ],
+    "time": "1 h 20 m",
+    "autolearn": true,
+    "qualities": [ { "id": "SEW", "level": 1 } ],
+    "tools": [ [ [ "sewing_kit", -1 ] ] ],
```

Comment:
Sewing kit isn't necessary; you're already using SEW quality. CUT quality would be also needed, as it's used for tailoring in general.